src/black/strings.py

```diff
@@ -18,6 +20,15 @@
     r"^([" + STRING_PREFIX_CHARS + r"]*)(.*)$", re.DOTALL
 )
 FIRST_NON_WHITESPACE_RE: Final = re.compile(r"\s*\t+\s*(\S)")
+UNICODE_RE: Final = re.compile(
```

Comment:
Please rename this variable more specific, maybe `UNICODE_ESCAPE_RE`?

Reply:
Done